pkg/plugins/discovery.go

```diff
@@ -37,25 +38,25 @@ func (l *LocalRegistry) Plugin(name string) (*Plugin, error) {
 	filepath := filepath.Join(l.path, name)
 	specpath := filepath + ".spec"
 	if fi, err := os.Stat(specpath); err == nil {
```

Comment:
I don't think this refactor is necessary yet. All this logic is changing in #1395